modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/praetorian.dm

```diff
@@ -0,0 +1,156 @@
+
+	icon_icon = 'modular_skyrat/modules/xenos_skyrat_redo/icons/xeno_actions.dmi'
+	button_icon_state = "throw_tail"
+
+	sparkle_path = /obj/effect/temp_visual/dir_setting/tailsweep/praetorian
+
+/datum/action/cooldown/spell/aoe/repulse/xeno/hard_throwing/IsAvailable()
+	. = ..()
+	if(!isalien(owner))
+		return FALSE
+	var/mob/living/carbon/alien/humanoid/skyrat/owner_alien = owner
+	if(owner_alien.unable_to_use_abilities)
+		return FALSE
+
+/datum/action/cooldown/spell/aoe/repulse/xeno/hard_throwing/cast_on_thing_in_aoe(atom/movable/victim, atom/caster)
+	if(isalien(victim))
+		return
+	var/turf/throwtarget = get_edge_target_turf(caster, get_dir(caster, get_step_away(victim, caster)))
+	var/dist_from_caster = get_dist(victim, caster)
+
+	if(dist_from_caster <= 0)
+		if(isliving(victim))
+			var/mob/living/victim_living = victim
+			victim_living.Knockdown(10 SECONDS)
+			victim_living.apply_damage(20, BRUTE,BODY_ZONE_CHEST, wound_bonus = 10) //It doesn't hurt as hard as other xenos because praetorians aren't meant to be frontlining
+			shake_camera(victim, 4, 3)
+			playsound(victim, 'sound/weapons/slap.ogg', 100, TRUE, 8, 0.9) //Ayo HOS this xeno ain't shi- SMACK
+			to_chat(victim, span_userdanger("You're slapped into the floor by [caster]'s tail!"))
+	else
+		if(sparkle_path)
+			new sparkle_path(get_turf(victim), get_dir(caster, victim))
+
+		if(isliving(victim))
+			var/mob/living/victim_living = victim
+			victim_living.Knockdown(4 SECONDS)
+			victim_living.apply_damage(20, BRUTE,BODY_ZONE_CHEST, wound_bonus = 10)
+			shake_camera(victim, 4, 3)
+			playsound(victim, 'sound/weapons/slap.ogg', 100, TRUE, 8, 0.9)
+			to_chat(victim, span_userdanger("[caster]'s tail crashes into you, throwing you back!"))
+
+		victim.safe_throw_at(throwtarget, ((clamp((max_throw - (clamp(dist_from_caster - 2, 0, dist_from_caster))), 3, max_throw))), 1, caster, force = repulse_force)
+
+/obj/effect/temp_visual/dir_setting/tailsweep/praetorian
+	icon = 'modular_skyrat/modules/xenos_skyrat_redo/icons/xeno_actions.dmi'
+	icon_state = "throw_tail_anim"
+
+/datum/action/cooldown/alien/acid/skyrat/spread
+	name = "Spit Neurotoxin Spread"
+	desc = "Spits a spread neurotoxin at someone, exhausting them."
+	plasma_cost = 50
+	acid_projectile = null
+	acid_casing = /obj/item/ammo_casing/caseless/xenospit
+	spit_sound = 'modular_skyrat/modules/xenos_skyrat_redo/sound/alien_spitacid2.ogg'
+	cooldown_time = 10 SECONDS
+
+/obj/item/ammo_casing/caseless/xenospit //This is probably really bad, however I couldn't find any other nice way to do this
+	name = "big glob of neurotoxin"
+	projectile_type = /obj/projectile/neurotoxin/skyrat/spitter_spread
+	pellets = 5
+	variance = 20
+
+/obj/item/ammo_casing/caseless/xenospit/tk_firing(mob/living/user, atom/fired_from)
+	return FALSE
+
+/obj/projectile/neurotoxin/skyrat/spitter_spread //Slightly nerfed because its a shotgun spread of these
+	name = "neurotoxin spit"
+	icon_state = "neurotoxin"
+	damage = 30
```

Comment:
That feels a bit strong, 5 pellets of 30 damage each, that's insta-crit at point-blank, no?

Reply:
i meant to put that back down to something else after I was doing testing, oops